public/configs/v1/env.json

```diff
@@ -35,11 +35,11 @@
             "image": "/currencies/usdc.png"
          }
       },
-      "[mainnet chain id]": {
-         "comment": "Change according to mainnet release",
+      "dydx-mainnet-1": {
```

Comment:
Guessing these are unintentionally included

Reply:
ah mannnn.  was having major dependency hell problems and had to do a lot of checking stuff out back and forth 😢